sample/F8/sessions

```diff
@@ -0,0 +1,1226 @@
+         "name":"CAMERA"
+      },
+      "end_time":"2017-04-18T13:20:00",
+      "start_time":"2017-04-18T12:30:00",
+      "speakers":[
+         {
+            "organisation":"Facebook",
+            "id":2,
+            "name":"Michael Slater"
+         }
+      ],
+      "microlocation":{
+         "id":5,
+         "name":"G"
+      }
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":7,
+      "subtitle":"litho-a-declarative-framework-for-efficient-UIs",
+      "title":"Litho: A Declarative Framework for Efficient UIs",
+      "sessionDescription":"Components for Android allows developers to implement complex, efficient RecyclerViews through a simple declarative API. In this session we’ll discuss the function and features of this powerful framework.",
+      "end_time":"2017-04-18T13:20:00",
+      "microlocation":{
+         "id":6,
+         "name":"F"
+      },
+      "start_time":"2017-04-18T12:30:00",
+      "comments": null,
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":1,
+         "name":"1 hour session",
+         "length":"01.00"
+      },
+      "track":{
+         "id":3,
+         "name":"ENGINEERING & OPEN SOURCE"
+      },
+      "speakers":[
+         {
+            "organisation":"Facebook",
+            "id":4,
+            "name":"Lucas Rocha "
+         },
+         {
+            "organisation":"Facebook",
+            "id":5,
+            "name":"Emil Sjölander"
+         }
+      ]
+   },
+   {
+      "short_abstract":null,
+      "long_abstract":null,
+      "id":8,
+      "subtitle":"webvr-enabling-developers-to-build-engaging-cross-platform-vr-content",
+      "title":"WebVR: Enabling Developers to Build Engaging Cross-Platform VR Content",
+      "sessionDescription":"Web-based frameworks such as React VR are making it easier than ever to create VR content that can be shared with a huge audience across VR, mobile, and the web. Coupled with the new web browser that Oculus is building, more of the web will become immersive over the coming years. Join us to learn what you can build with WebVR and how this new technology can help enhance your existing content.",
+      "start_time":"2017-04-18T12:30:00",
+      "end_time":"2017-04-18T13:20:00",
+      "microlocation":{
+         "id":2,
+         "name":"LL"
+      },
+      "comments": null,
+      "language":null,
+      "slides":null,
+      "audio":null,
+      "video":null,
+      "signup_url":null,
+      "state":"accepted",
+      "timezone":"UTC",
+      "session_type":{
+         "id":1,
+         "name":"1 hour session",
+         "length":"01.00"
+      },
+      "track":{
+         "id":5,
+         "name":"VIRTUAL REALITY"
+      },
+      "speakers":[
+         {
+            "organisation":"Facebook",
+            "id":6,
+            "name":"Justin Rogers"
```

Comment:
Yeah, I just saw the extra fields in your sample however, I didn't see it any other sample. How did you come about all of this information?